### PR TITLE
#8: Implemented Statistic Object creation

### PR DESCRIPTION
**Pull request by mperrinel**
This PR aims to provide a statistic object that can be manipulated from the python file.
More information are available [here](https://github.com/mperrinel/sst-core/wiki/Statistic-Python-Object).

### CODE REVIEW

**Comment by jjwilke** [comment]
We should take care of this PR first:
https://github.com/sstsimulator/sst-core/pull/540

Then we can avoid replicating a bunch of Python code. Labeling this WIP.

**Comment by mperrinel** [comment]
Hi Allevin, thanks for the review.
The new files (pymodel_stat.cc, pymodel_stat.h) are included in the **core/model/Makefile.inc**, in the **sst_core_python3_sources** variable. This variable is included into the Makefile.am file :
```
sst_core_sources += $(sst_core_python3_sources)
```

**Comment by mperrinel** [comment]
@jjwilke this PR has been updated accordingly to the #540.

**Comment by mperrinel** [comment]
@jjwilke 
This PR has been rebased on the latest devel.

File: src/sst/core/componentInfo.h [hunk 1]
```diff
@@ -26,6 +26,13 @@ class LinkMap;
 class BaseComponent;
 
 class ConfigComponent;
+
+namespace EXPERIMENTAL {
```
[the remark this answers is not in the document]
Comment (mperrinel) [reply src/sst/core/componentInfo.h]: done 👍 a30a56f151c1c585226c189580d2fc5cd7e0ee1e

**Pull request closed without merging**